.travis.yml

```diff
@@ -0,0 +1,30 @@
+sudo: required
+dist: trusty
+language: go
+go:
+  - "1.11.x"
+
+matrix:
+  include:
+    - os: linux
+      env: VERSION_UPGRADE_TEST_WAIT_TIMEOUT=30s
+    - os: osx
+      # Do not start osx build for PR
+      if: type != pull_request
+      osx_image: xcode8
+      env: VERSION_UPGRADE_TEST_WAIT_TIMEOUT=30s
+
+install:
+  - cd $GOPATH/src/github.com/denisacostaq/rextporter
```

Comment:
No . Use upstream repository name . See [`skycoin/skycoin` .travis.yml](https://github.com/skycoin/skycoin/blob/75ec497/.travis.yml) for a reference , [line 47](https://github.com/skycoin/skycoin/blob/75ec497/.travis.yml#L47) in particular .